tests/manual_performance_test/compare.py

```diff
@@ -0,0 +1,97 @@
+import statistics
```

Comment:
Don't you want to make this comparison as normal / standard test? The assert could be that it is faster :)

Reply:
Automated performance tests are so much work to make stable that I don't feel that it's worth it
I can run the manual ones every now and then to see if I mess something up :)